model/enduser/common.yaml

```diff
@@ -0,0 +1,22 @@
+groups:
+  - id: identity
+    type: attribute_group
+    brief: >
+      These attributes may be used for any operation with an authenticated and/or authorized enduser.
+    attributes:
+      - ref: enduser.id
+        deprecated: Replaced by `enduser.pseudo.id` attribute.
+        requirement_level: recommended
+      - ref: enduser.pseudo.id
+        requirement_level: recommended
+      - ref: enduser.role
```

Comment:
please remove it from this group - it's listed as deprecated attribute and is not necessary here